soc/arm/st_stm32/common/soc_config.c

```diff
@@ -60,13 +60,47 @@ static int st_stm32_common_config(void)
 	LL_APB2_GRP1_EnableClock(LL_APB2_GRP1_PERIPH_DBGMCU);
 #endif /* LL_APB1_GRP1_PERIPH_DBGMCU */
 
+#endif /* CONFIG_USE_SEGGER_RTT */
+
+
+#if defined(CONFIG_STM32_ENABLE_DEBUG_SLEEP_STOP)
+
 #if defined(CONFIG_SOC_SERIES_STM32H7X) || defined(CONFIG_SOC_SERIES_STM32MP1X)
-	HAL_EnableDBGSleepMode();
-#else
+	HAL_EnableDBGStopMode();
+#else /* CONFIG_SOC_SERIES_STM32H7X || CONFIG_SOC_SERIES_STM32MP1X */
+#if defined(SOC_SERIES_STM32G0X)
+	LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_DBGMCU);
+	LL_DBGMCU_EnableDBGStopMode();
+	LL_APB1_GRP1_DisableClock(LL_APB1_GRP1_PERIPH_DBGMCU);
+#elif defined(SOC_SERIES_STM32L0X)
+	LL_APB2_GRP1_EnableClock(LL_APB2_GRP1_PERIPH_DBGMCU);
+	LL_DBGMCU_EnableDBGStopMode();
+	LL_APB2_GRP1_DisableClock(LL_APB2_GRP1_PERIPH_DBGMCU);
+#else /* all other parts */
```

Comment:
STM32C0 and F0 also have a clock for DBGMCU (respectively LL_APB1_GRP1_PERIPH_DBGMCU and LL_APB1_GRP2_PERIPH_DBGMCU).
I know that PM is not implemented for these series, but wouldn't it be best to add these two cases nevertheless?